libwild/src/args.rs

```diff
@@ -838,6 +825,15 @@ impl ArgumentParser {
         arg: &str,
         input: &mut I,
     ) -> Result<()> {
+        // Handle `@file`option (recursively) - merging in the options contained in the file
+        if let Some(path) = arg.strip_prefix('@') {
+            let file_args = read_args_from_file(Path::new(path))?;
+            let mut file_arg_iter = file_args.iter();
+            while let Some(file_arg) = file_arg_iter.next() {
+                self.handle_argument(args, modifier_stack, file_arg, &mut file_arg_iter)?;
+            }
+        }
```

Comment:
It looks odd to me that there's not a return here. Won't that mean that we'll do something else with `arg` even after we read the response file?

Reply:
Fixed, now it matches the other branches